AAP2336 - Created topics for Migration & Upgrade of Operator on OCP (#460)
RedHatInsights/red-hat-ansible-automation-platform-documentation: downstream/modules/platform/proc-aap-migration.adoc

[id="aap-migration_{context}"]

= Migrating data to the {PlatformNameShort} Operator

[role=_abstract]

> 

Reviewer: Is there just one step to this procedure? It almost doesn't read like a procedure. Seems more conceptual with the way it's written. See suggestion below.

> Once you have set your secret key, postgresql credentials, verified network connectivity and installed the {OperatorPlatform}, you can migrate your data by creating a custom resource controller object.

Reviewer: ```suggestion
.Prerequisites

* You have set your secret key. 
* You have set your postgresql credentials. 
* You have verified network connectivity. 
* You have installed the {OperatorPlatform}. 

.Procedure

. Migrate your data by creating a custom resource controller object. 
```
This suggestion is based on [Modular Documentation standard](https://redhat-documentation.github.io/modular-docs/)s for procedure modules. The question now is will the end user know how to create a custom resource controller object? Is this something that is assumed for average end user who will be following this procedure? Are there steps that are missing?

> Once you have set your secret key, postgresql credentials, verified network connectivity and installed the {OperatorPlatform}, you can migrate your data by creating a custom resource controller object.

Reviewer: Suggestion: Change to imperative voice to fit procedure module (task-oriented) like the following:
 ```suggestion
After you have set your secret key, postgresql credentials, verified network connectivity and installed the {OperatorPlatform}, migrate your data by creating a custom resource controller object.
```

Author: Good call! Fixed.
